#!/usr/bin/env python
PACKAGE = "plane_slam"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

##
gen.add("do_visual_odometry", bool_t, 0, "", False)
gen.add("do_mapping", bool_t, 0, "", False)
gen.add("do_slam", bool_t, 0, "", True)
gen.add("map_frame", str_t, 0, "", "/map")
gen.add("base_frame", str_t, 0, "", "/base_footprint")
gen.add("odom_frame", str_t, 0, "", "/world")
gen.add("skip_message", int_t, 0, "", 1, 1, 30)
##

exit(gen.generate(PACKAGE, "plane_slam", "PlaneSlam"))
